style.css

```diff
@@ -321,3 +324,98 @@ textarea::placeholder {
   color: #fff;
   border-color: #2230d2;
 }
+
+@media (min-width: 992px) {
+  header {
+    padding: 24px 155px;
+  }
+
+  .nav-links {
+    display: flex;
+    gap: 32px;
+    list-style-type: none;
+  }
+
+  .nav-links a {
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .nav-links a:hover {
+    color: #6070ff;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .hero {
+    padding: 260px;
```

Comment:
please next time you can leave this comment as optional because it is just like the Figma design.

![image](https://user-images.githubusercontent.com/42470159/144460153-e358b921-4d68-4628-938a-aed52dd59302.png)